libparsec/crates/client/src/invite/claimer.rs

```diff
@@ -207,9 +218,10 @@ async fn run_claimer_step_until_ready(
 }
 
 #[derive(Debug)]
-pub enum UserOrDeviceClaimInitialCtx {
+pub enum AnyClaimRetrievedInfoCtx {
     User(UserClaimInitialCtx),
     Device(DeviceClaimInitialCtx),
+    ShamirRecovery(ShamirRecoveryClaimPickRecipientCtx),
```

Comment:
So in the end no `ShamirRecoveryNonRecoverable(ShamirRecoveryNonRecoverableCtx)` variant, we stick with the `is_recoverable` method instead ?

Reply:
Yes, easier this way